psdisk/psdisk.c

```diff
@@ -129,8 +116,9 @@ static void psdisk_showPartsTable(void)
 	printf("\n");
 	printf(BOLDWHITE PARTITION_HEADER_ALIGMENT RESET, "Name", "Start", "End", "Blocks", "Size", "Type");
 	LIST_FOREACH(node, &psdisk_common.list, ptrs) {
+		type = ptable_typeName(node->part.type);
 		printf(PARTITION_DATA_ALIGMENT, node->part.name, node->part.offset, node->part.offset + node->part.size,
-			   node->part.size / psdisk_common.blksz, node->part.size, psdisk_getTypeName(node->part.type));
+				node->part.size / psdisk_common.blksz, node->part.size, (type != NULL) ? type : "???");
```

Comment:
not related to this PR - but I would like to know why we're printing partitions in ~~retarded~~ reverse order :) :
![image](https://github.com/user-attachments/assets/aee74645-865a-46f5-bb50-1740544e122c)